src/pages/Home/homepage.jsx

```diff
@@ -1,81 +1,106 @@
+import {home1,home2,home3,banner1,banner2,banner3,bannerV,brand1,brand2,brand3,brand4,brand5} from "../../Assets/index"
 
 
 const Home = () => {
 
-    const[display, setDisplay] = useState("men")
 
-    const MenDisplay = () =>   <div className="sliding-cards">
-    <div>
-    <img src= {men1} alt="" />
 
-    </div>
-    <div>
-    <img src= {men2} alt="" />
-
-    </div>
-    <div>
-    <img src= {men3} alt="" />
-    </div>
-    </div>
-
-    const WomenDisplay = () =>   <div className="sliding-cards">
-    <div>
-    <img src={women2} alt="" />
-    </div>
-    <div>
-    <img src= {women1} alt="" />
-    </div>
-    <div>
-    <img src= {women3} alt="" />
-    </div>
-    </div>
-
-    const LuxuryDisplay = () =>   <div className="sliding-cards">
-    <div>
-    <img src= {luxury1} alt="" />
-    </div>
-    <div>
-    <img src= {luxury2} alt="" />
-    </div>
-    <div>
-    <img src= {luxury3} alt="" />
-    </div>
-    </div>
   
   return (
     <>              
     {/* ------------------------------------bANNER */}
+              <div className="banner-boxy col-12">
+              <div className="banner-text col-6">
+              <div className="banner-heading"> <span className="banner-design-text"> The Perfect Moment </span>Between Past And Future</div>
+              <div className="banner-para">We'have had a long standing mission of providing unique and affordable  timepieces to adventurous enthusiasts as yourself</div>
+             <Link to ="/products"><button className="banner-button ">Explore Collection</button></Link>
+              </div>
+              <div className="banner-images col-6">
+                <div className="banner-image-1 col-5">
+                  <img src={banner3} alt="" />
+                  
+                </div>
+                <div className="banner-image-1 col-5">
+                <img src={banner2} alt="" />
+
+                </div>
+                <div className="banner-image-1 col-2">
+                <img src={banner1} alt="" />
+
+                </div>
+              </div>
+                </div>
+           {/*---------------------------------------------  */}
+        <div className="banner-headline col-12">
+          <img src={bannerV} alt="" />
+          {/* ------------------------------------------------- */}
 
-            <div className="banner-box col-12">
-            <Link to="/products" > <img src={banner1} alt=""/></Link>
+          <div className="cards-outer-box">
+            <div className="headline">Trending Items</div>
+            <div className="cards-box">
+            
+            <div className="card">
+              <div className="card-img-box">
+           <img src= "https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/9460049/2019/10/5/c47398e2-72a5-45a4-bcb5-dd58adee9b161570274183766-Roadster-Men-Black-Multi-Function-Analogue-and-Digital-Watch-1.jpg" alt="" />
```

Comment:
This huge image source looks messy to me in between the main file's code. you can store them separately.